vars/tasks.groovy

```diff
@@ -91,10 +91,10 @@ def functionalXpack(Map params = [:]) {
   ]
 
   task {
-    kibanaPipeline.buildXpack(10)
+    kibanaPipeline.buildXpack(11)
```

Comment:
This one should stay 10. It's just the number of parallel workers for building Kibana, not related to ciGroups.

Reply:
I've reset the value to the previous number